src/tests/ratingController.spec.ts

```diff
@@ -122,6 +174,48 @@ describe('Rating controller tests', () => {
 		expect(res.statusCode).to.equal(404);
 	});
 
+	it('Get top five average ratings returns 200', async () => {
+		const req = mockReq();
+		const res = mockRes({
+			status: function(s: number) {
+				this.statusCode = s;
+				return this;
+			},
+		});
+		sinon.stub(RatingService, 'getTopFiveAverageRatings').resolves(mockTopFiveAverageRatings);
+		await RatingController.getTopFiveAverageRatings(req, res);
+		expect(res.statusCode).to.equal(200);
+	});
+
+	it('Should properly call the top five average rating function', async () => {
+		const req = mockReq();
+		const res = mockRes({
+			status: function() {
+				return {
+					send: (e: object[]): object[] => {
+						return e;
+					},
+				};
+			},
+		});
+		sinon.stub(RatingService, 'getTopFiveAverageRatings').resolves(mockTopFiveAverageRatings);
+		const topFiveAverageRatings = await RatingController.getTopFiveAverageRatings(req, res);
+		expect(topFiveAverageRatings).equals(mockTopFiveAverageRatings);
+	});
+
+	it('Shoudl fail to get top five average ratings and return 404', async () => {
```

Comment:
```suggestion
	it('Should fail to get top five average ratings and return 404', async () => {
```

måske skrive at det er hvis servicen failer